src/tests/contract_test/wallet_test.cljs

```diff
@@ -0,0 +1,84 @@
+(ns tests.contract-test.wallet-test
+  (:require
+    [cljs.test :refer [deftest is]]
+    [day8.re-frame.test :as rf-test]
+    legacy.status-im.events
+    [legacy.status-im.multiaccounts.logout.core :as logout]
+    legacy.status-im.subs.root
+    [status-im.common.emoji-picker.utils :as emoji-picker.utils]
+    [status-im.contexts.wallet.data-store :as data-store]
+    status-im.events
+    status-im.navigation.core
+    status-im.subs.root
+    [test-helpers.integration :as h]
+    [tests.contract-test.utils :as contract-utils]))
+
+(defn assert-accounts-get-accounts
+  [result]
+  (is (true? (some :wallet result)))
+  (is (true? (some :chat result)))
+  (is (= 2 (count result))))
+
+(deftest accounts-get-accounts-contract
+  (h/log-headline :contract/accounts-get-accounts)
+  (rf-test/run-test-async
+   (h/with-app-initialized
+    (h/with-account
+     (contract-utils/call-rpc-endpoint
+      {:rpc-endpoint "accounts_getAccounts"
+       :action       assert-accounts-get-accounts})
+     (h/logout)
+     (rf-test/wait-for
+       [::logout/logout-method])))))
+
+(defn get-default-account
+  [accounts]
+  (first accounts))
+
+(defn check-emoji-is-updated
+  [test-emoji]
+  (fn [accounts]
+    (let [default-account (first accounts)]
+      (is (= (:emoji default-account) test-emoji)))))
+
+(deftest accounts-save-accounts-contract
```

Comment:
this test has slight more steps.
1. get default wallet account using `accounts_getAccounts`
2. update default wallet account emoji
3. assert emoji updated